server/repository/userRepository.js

```diff
@@ -50,29 +68,44 @@ class UserRepository {
   }
 
   /**
- * Finds a user by email and username
- * @param {string} email Email to search by
- * @param {string} username username
- * @returns {object | null} User object or null if user is not found
- */
+* Finds a user by parameter
+* @param {string} param Parameter to search by
+* @param {string} value Parameter value
+* @returns {object | null} User object or null if user is not found
+*/
+  static async getUserByParam(param, value) {
+    if (param === 'email') {
+      attributes.push('password');
```

Comment:
why?

Reply:
@seyi-adeleke the login route needs to check that passwords match.